prometheus/src/token.rs

```diff
@@ -0,0 +1,196 @@
+use std::{
```

Comment:
This is a nice utility from Solido, but it looks like overkill for what we need here. I think just a `struct Lamports(pub u64)` would also suffice.